### PR TITLE
[Quick Add Bulk] Quick fix for the Progress bar

### PR DESCRIPTION
**Pull request by eugenekasimov**
### PR Summary: 

There is an issue happened after refactoring in the Progress Bar PR https://github.com/Shopify/dawn/pull/3288/commits/9b49ae369183667f25a143420954884722817546

### Why are these changes introduced?

The issue is that before we moved `this.querySelector('quantity-input input')` into a variable the input element was re-selected every time the input changed and it worked good. Since we store it in a variable `this.input `we select the input once but for DOM it’s different element every time we change qty on it.

### What approach did you take?
I added a function to set input every time after we change qty on the input. 

### Demo links

- [Editor](https://admin.shopify.com/store/os2-demo/themes/163009855510/editor)

### Checklist
- [ ] Added PR summary for [release notes](https://themes.shopify.com/themes/dawn/styles/default#ReleaseNotes)
- [ ] Requested review from UX (Only for changes that are affecting the experience or perceivable visual details)
- [ ] Created a ticket for the [help.shopify.com](https://help.shopify.com) documentation team about updates to theme settings. (Internal-only task)
- [ ] Followed [theme code principles](https://github.com/Shopify/dawn/blob/main/.github/CONTRIBUTING.md#theme-code-principles)
- [ ] Linted with [Theme Check](https://github.com/Shopify/theme-check)
- [ ] Tested on [mobile](https://shopify.dev/themes/store/requirements#mobile-browser-requirements)
- [ ] Tested on [multiple browsers](https://shopify.dev/themes/store/requirements#desktop-browser-requirements)
- [ ] Tested for [accessibility](https://shopify.dev/themes/best-practices/accessibility)

### CODE REVIEW

File: assets/quick-add-bulk.js [hunk 1]
```diff
@@ -33,6 +33,11 @@ class QuickAddBulk extends HTMLElement {
     }
   }
 
+  setInput() {
+    this.input = this.querySelector('quantity-input input');
+    return this.input;
```
Comment (sofiamatulis) [inline assets/quick-add-bulk.js:38]: Do we need to return this? We already saved it to a variable

File: assets/quick-add-bulk.js [hunk 2]
```diff
@@ -10,7 +10,7 @@ class QuickAddBulk extends HTMLElement {
       }
     }, ON_CHANGE_DEBOUNCE_TIMER);
     this.addEventListener('change', debouncedOnChange.bind(this));
-    this.input = this.querySelector('quantity-input input');
+    this.setInput();
```
[the remark this answers is not in the document]
Comment (sofiamatulis) [reply assets/quick-add-bulk.js:13]: thanks for clarifying! I didnt remember if we had it in the 1st commit 👍

**Review by sofiamatulis — APPROVED** [review]

File: assets/quick-add-bulk.js [hunk 3]
```diff
@@ -33,6 +33,10 @@ class QuickAddBulk extends HTMLElement {
     }
   }
 
+  setInput() {
+    this.input = this.querySelector('quantity-input input');
```
Comment (utkarshsaxena93) [inline assets/quick-add-bulk.js:37]: Can we do 
```
getInput() {
    return this.querySelector('quantity-input input');
}
```

and then do `this.getInput().addEvent.....` or wherever else it is used.

Why do we want to store it in a variable?

Comment (eugenekasimov) [reply assets/quick-add-bulk.js]: Thanks @utkarshsaxena93 I need a bit time to learn this approach and figure out how it works but looks interesting even for my development. I'll take a look :)

**Pull request merged**